src/calculated-property-rule.ts

```diff
@@ -67,7 +71,29 @@ export class CalculatedPropertyRule extends Rule {
 			this.property.isCalculated = true;
 	}
 
-	execute(obj: Entity): void {
+	register() {
+		super.register();
+
+		// TODO: Restrict this to default values or apply to all calculation?
+		// Why is this here?
+		if (this.isDefaultValue) {
```

Comment:
Removing this check definitely breaks tests.

Reply:
Yeah, not sure why I was doubting the need for that. 🤷‍♂️